chunks/modelevaluation.ipynb

```diff
@@ -0,0 +1,935 @@
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Further reading\n",
+    "\n",
+    "* Mackay, Chapters 3 and 28\n",
+    "\n",
+    "* Gelman et al, Chapters 6 and 7"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Model Evaluation\n",
+    "\n",
+    "* You can't do inference without making assumptions.\n",
+    "\n",
+    "* We must _test_ the hypotheses defined by our models.\n",
+    "\n",
+    "\n",
+    ">  Does the model provide an _accurate_ description of the data?\n",
+    ">  \n",
+    ">\n",
+    ">  Are there alternative models that are either more accurate, more efficient, or both?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Model Checking\n",
+    "\n",
+    "* How do we know if our model is any good? One property that \"good\" models have is *accuracy.*\n",
+    "\n",
+    "\n",
+    "* Accurate models generate data that is *like* the observed data. What does this mean? \n",
+    "\n",
+    "\n",
+    ">  * A model's ability to \"predict the data\" can often be assessed _visually_\n",
+    "> \n",
+    ">\n",
+    ">  * *Test statistics* can be designed to capture discrepancies between observed and predicted data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Model Checking Example\n",
+    "\n",
+    "We'll step through a simple [example analysis](../examples/Straightline), of some $(x, y, \\sigma)$ data drawn generated by a simple linear model:\n",
+    "\n",
+    "<img src=\"../graphics/modelcheck-data.png\" width=60%>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Fitting a Straight Line $y = m x + b$\n",
+    "\n",
+    "<img src=\"../graphics/modelcheck-linear-posterior.png\" width=50%> This looks like a nice, precise measurement."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Visual Check\n",
+    "\n",
+    "* Does the model fit the data?\n",
+    "\n",
+    "* The first thing to do is plot the model's predictions, in data space\n",
+    "\n",
+    "* In this example, \"realizing the model\" means drawing lines of $m x + b$ through the data points\n",
+    "\n",
+    "* We have posterior samples in $(m, b)$: let's realize them all, with low `alpha` to see where the probability mass is"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Visual Check\n",
+    "\n",
+    "<img src=\"../graphics/modelcheck-linear.png\" width=60%>"
```

Comment:
I wonder whether we should introduce more sophisticated visualizations like quantile-quantile plots